### ISSUE TITLE
Tab Completion does not work for windows 11 on PowerShell Core, like it literally will not work

### ISSUE THREAD

**Issue by kris3713** [liuzijing97/scoop-auto-completion#34]
Sorry for the bad video quality,

https://github.com/Moeologist/scoop-completion/assets/101912712/d8e76b89-6f8e-46f5-a7e7-ba313451399a

**Comment by Lockszmith-GH** [comment]
After you installed the scoop-completion with `scoop install scoop-completion` did you run the `Import-Module` as described in the (README)[https://github.com/Moeologist/scoop-completion#scoop-completion]?

```bash
# enable completion in current shell, use absolute path because PowerShell Core not respect $env:PSModulePath
Import-Module "$($(Get-Item $(Get-Command scoop.ps1).Path).Directory.Parent.FullName)\modules\scoop-completion"
```

PowerShell TabExpansion relies on a module named PSReadLine, is it installed?

Please provide the output of the following commands:

$PSVersionTable
Get-Modules
Get-PSReadLineKeyHandler

This information should be helpful in pin pointing possible cause.

**Comment by maoyeedy** [comment]
This is broken due to pwsh 7.4.0 upgrade, scoop-completion works fine in 7.3.10

**Comment by seanmamasde** [comment]
yes as @Maoyeedy said, it's broken since the update. Any update on this issue?

**Comment by kris3713** [comment]
> This is broken due to pwsh 7.4.0 upgrade, scoop-completion works fine in 7.3.10

So I'll just downgrade to a previous version of PowerShell. Works fine for me.

**Comment by kris3713** [comment]
It works now for me:
![(WindowsTerminal)k482f7_11-18-2023](https://github.com/Moeologist/scoop-completion/assets/101912712/671f8339-213b-4776-b1dd-9ed87c37bafd)

**Comment by kris3713** [comment]
@Maoyeedy so this bug is just a PowerShell related one, not a PS module one?

**Comment by nahid6970** [comment]
not working for me either on  Powershell 7.4

**Comment by kris3713** [comment]
> not working for me either on  Powershell 7.4

The current solution is to downgrade to PowerShell version 7.3.10, unfortunately.

**Comment by nahid6970** [comment]
any solution found guys?

**Comment by seanmamasde** [comment]
How is this solved? downgrading to make it work does not count as a solution.

@Zliced13 You should reopen this issue for better visibility, as it is still unsolved. Downgrading is just a temporary workaround, you won't be using pwsh ver7.3.10 forever. Thank you.

**Comment by kris3713** [comment]
@seanmamasde Sorry for closing this issue report 3 days ago, I was just annoyed by the spammy notifications.

EDIT: If this issue is resolved, could someone please mention me in a message?

**Comment by seanmamasde** [comment]
@Zliced13 If you don't want message notifications, you could simply unsubscribe from this issue, so you won't get it. I am not sure whether the issue you opened gets closed you'll get notified or not, but sure if I remember I will mention you so you'll know.

**Comment by kris3713** [comment]
> @Zliced13 If you don't want message notifications, you could simply unsubscribe from this issue, so you won't get it. I am not sure whether the issue you opened gets closed you'll get notified or not, but sure if I remember I will mention you so you'll know.

I already unsubscribed from this thread @seanmamasde

**Comment by Moeologist** [comment]
Fixed on v0.3.0

**Comment by kris3713** [comment]
@Moeologist thank you for fixing that bug
